scripts/utils/envutils.py

```diff
@@ -0,0 +1,138 @@
+import os
+from pathlib import Path
+
+LOGLEVEL_MAP = {
+    'none': (0, 'LOG_NONE'),
+    'log_none': (0, 'LOG_NONE'),
+    'error': (1, 'LOG_ERROR'),
+    'log_error': (1, 'LOG_ERROR'),
+    'warn': (2, 'LOG_WARN'),
+    'log_warn': (2, 'LOG_WARN'),
+    'warning': (2, 'LOG_WARN'),
+    'log_warning': (2, 'LOG_WARN'),
+    'info': (3, 'LOG_INFO'),
+    'log_info': (3, 'LOG_INFO'),
+    'debug': (4, 'LOG_DEBUG'),
+    'log_debug': (4, 'LOG_DEBUG'),
+    'verbose': (5, 'LOG_VERBOSE'),
+    'log_verbose': (5, 'LOG_VERBOSE'),
+}
+
+
+def __bool(key: str, value: str | None) -> bool:
+    if value == None:
+        return False
+
+    value = value.lower()
+
+    if value == 'true' or value == '1':
+        return True
+    elif value == 'false' or value == '0':
+        return False
+    else:
+        raise ValueError('Environment variable ' + key + ' is not a boolean value.')
+
+
+def get_bool(key: str) -> bool:
+    return __bool(key, os.environ.get(key, None))
+
+
+def get_github_ref_name():
+    return os.environ.get('GITHUB_REF_NAME', None)
+
+
+def get_github_base_ref():
+    return os.environ.get('GITHUB_BASE_REF', None)
+
+
+def get_github_event_name():
+    return os.environ.get('GITHUB_EVENT_NAME', None)
```

Comment:
not a fan of this; you have this `get_bool` method for getting bools but you don't have a generic `get_string` for these?

also, there's no need for any of these specific env vars to have a dedicated method to begin with